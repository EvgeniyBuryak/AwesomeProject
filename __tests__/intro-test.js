import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../src/screens/intro/intro.view';

test('renderer correctly', () => {
  const tree = renderer.create(<Intro item={'React'} />).toJSON();

  expect(tree).toMatchSnapshot();
});
