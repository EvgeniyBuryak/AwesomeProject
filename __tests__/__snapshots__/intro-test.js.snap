// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#333",
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to city 17!
  </Text>
  <Text
    style={
      Object {
        "color": "#333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    This is a 
    React
     Native snapshot test.
  </Text>
</View>
`;
